modules/users/server/config/users.server.config.js

```diff
@@ -11,7 +11,7 @@ var passport = require('passport'),
 /**
  * Module init function
  */
-module.exports = function (app, db) {
+module.exports = function (app) {
```

Comment:
I like that, sounds simple and straightforward. I didn't like at all trying to follow where `db` is going around trough all those callbacks.

That said, this could also wait for more thorough restructuring of mongoose/db stuff as with promises etc.